src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -156,23 +156,48 @@ public ExchangeService(BidderCatalog bidderCatalog,
 
         final long startTime = clock.millis();
 
-        return extractBidderRequests(bidRequest, uidsCookie, aliases, timeout)
+        // sanity check: discard imps without extension
+        final List<Imp> imps = imps(bidRequest);
+
+        final Map<String, List<String>> deprecatedBiddersErrors = new HashMap<>();
```

Comment:
Rename pls to `errors` because it can be global map for error through processing workflow, for ex. i plan to add cache service errors.